src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/Generated/Abgr32.PixelOperations.Generated.cs

```diff
@@ -57,7 +57,7 @@ public override void ToAbgr32(Configuration configuration, ReadOnlySpan<Abgr32>
         {
             Vector4Converters.RgbaCompatible.ToVector4(configuration, this, sourcePixels, destVectors, modifiers.Remove(PixelConversionModifiers.Scale));
         }
-    
```

Comment:
Any idea what caused these changes? 

We should probably add #nullable disable to all T4 templates at some point.

Reply:
No clue. I reverted a bunch of changes. Probably overlooked this one. 

I can only imagine that opening the file in rider on Mac caused this.